src/AssetRegistry/ATokenPricingModule.sol

```diff
@@ -163,5 +249,9 @@ contract ATokenPricingModule is PricingModule {
                 rateInUsd, assetToInformation[getValueInput.assetAddress].assetUnit
             );
         }
+
+        collFactor = assetToInformation[getValueInput.assetAddress].assetCollateralFactors[getValueInput.baseCurrency];
```

Comment:
Now we fetch 5 times getValueInput.assetAddress, still not cheaper to take assetAddress in to memory?